layers/+lang/dart/packages.el

```diff
@@ -13,8 +13,11 @@
 
 ;; TODO: offer lsp integration
 (defconst dart-packages
-  '(dart-mode
-    flycheck))
+  '(
+    dart-mode
+    (dart-server :location (recipe :fetcher github :repo bradyt/dart-server))
+    flycheck
+    ))
 
```

Comment:
Shouldn't this be a one line addition? If I understand correctly, parens on empty lines is not idiomatic.

The trailing parens seem to be explicitly discouraged. See:

> Don't make a habit of putting close-parentheses on lines by themselves; Lisp programmers find this disconcerting.

-- https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html

Reply:
I tried it without the surrounding parentheses first (as you suggested in the related issue) but Spacemacs shows these warnings on startup:
```
Warnings:
    - package :location not initialized in layer dart, you may consider removing this
      package from the package list or use the :toggle keyword instead of a `when'
      form.      
    - package recipe not initialized in layer dart, you may consider removing this
      package from the package list or use the :toggle keyword instead of a `when'
      form.      
```

The spaces between packages and keywords seem to work as package delimiters.

The `layers.org` documentation shows that packages that use keywords are surrounded by parentheses.
https://github.com/syl20bnr/spacemacs/blob/develop/doc/LAYERS.org#packagesel

The example also uses trailing parentheses.

That might be why most of the Spacemacs layers used that style.
I unified all layers to use the same style:
Unify packages list parentheses style #11619

But that style seems to be limited to the `<layer>-packages` list, otherwise the Emacs convention seems to be encouraged.